CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+## [2.3.1] - 2017-10-19
+### Added
+- Updated silenced api endpoints
```

Comment:
I think we should give a bit more information to users about this change and the impact it has. I am thinking something along these lines:

Updated silenced api endpoints for clients greater than 0.26 If you are using < 0.26 then you will need to pin this version and only use plugins that rely on this version or lower.